cmd/kubeadm/app/constants/constants.go

```diff
@@ -366,6 +366,34 @@ const (
 	// May be overridden by a flag at startup.
 	// Deprecated: use the secure KubeControllerManagerPort instead.
 	InsecureKubeControllerManagerPort = 10252
+
+	// IPVSProxyMode is match set up cluster with ipvs proxy model
+	// TODO (yastij): we should make this shared also with componentConfig
+	IPVSProxyMode = "ipvs"
+
+	// ModeAlwaysAllow is the mode to set all requests as authorized
+	// TODO (yastij): we should make this shared also with componentConfig
+	ModeAlwaysAllow = "AlwaysAllow"
```

Comment:
Use [KubeletAuthorizationModeAlwaysAllow](https://github.com/kubernetes/kubelet/blob/master/config/v1beta1/types.go#L749)?

Reply:
hmmm nvm, ignore that.. don't love the inconsistency

Reply:
@liggitt @dims do you see a better home for `pkg/kubeapiserver/authorizer/modes`? Maybe `k8s.io/apiserver` or is that not generic enough?